test/SourceCall/unresolved.C

```diff
@@ -6,7 +6,6 @@
 // LICENSE.TXT for details.
 //------------------------------------------------------------------------------
 
-// RUN: cat %s | %cling
```

Comment:
In the past we used to have issues when cling crashed, FileCheck could still return success. This is a check against cling crashing.